aruna/api/storage/services/v1/collection_service.proto

```diff
@@ -107,6 +107,19 @@ service CollectionService {
       body : "*"
     };
   }
+
+  // AddKeyValueToCollection
+  //
+  // Status: BETA
+  //
+  // Adds key values (labels / hooks) to a collection
+  rpc AddKeyValueToCollection(AddKeyValueToCollectionRequest)
```

Comment:
This should be named as `AddKeyValuesToCollection` because more than one KeyValue-Pair can be added.

Reply:
Fixed by [24fc061](https://github.com/ArunaStorage/ArunaAPI/pull/79/commits/24fc061dd1efcc18cf4e1bd68ca04c792e52b35d)